dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -380,18 +411,14 @@ URL findClassLocation(String className) throws ClassNotFoundException {
     com.google.common.reflect.ClassPath classPath =
         com.google.common.reflect.ClassPath.from(classLoaderFromJar);
 
-    // Nested (inner) classes reside in one of top-level class files.
-    ImmutableSet<ClassInfo> allClassesInJar = classPath.getTopLevelClasses();
-    return allClassesInJar;
+    return classPath.getAllClasses();
   }
 
-  private static ImmutableSet<JavaClass> topLevelJavaClassesInJar(Path jarFilePath)
-      throws IOException {
-    String pathToJar = jarFilePath.toString();
-    SyntheticRepository repository = SyntheticRepository.getInstance(new ClassPath(pathToJar));
+  private static ImmutableSet<JavaClass> allJavaClassesInJar(Path jarFilePath) throws IOException {
```

Comment:
jarFilePath --> jarPath
allJavaClassesInJar --> getClassesInJar (We can assume there are no non-Java classes there, and method names are usually verbs.)